spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/controller/BaseController.java

```diff
@@ -101,10 +107,20 @@ protected Context parseOriginatingIdentity(String originatingIdentityString) {
 
 		String platform = parts[0];
 
-		return Context.builder()
-				.platform(platform)
-				.properties(properties)
-				.build();
+		if (CLOUD_FOUNDRY_PLATFORM.equals(platform)) {
```

Comment:
Mind if we do `CloudFoundryContext.CLOUD_FOUNDRY_PLATFORM.equals()` instead of using static imports? Same for